src/lib/heliaFetch.ts

```diff
@@ -1,14 +1,13 @@
 import { createVerifiedFetch, type ContentTypeParser } from '@helia/verified-fetch'
 import { fileTypeFromBuffer } from '@sgtpooki/file-type'
-import { dnsLinkLabelDecoder, isDnsLabel } from './dns-link-labels.ts'
 import type { Helia } from '@helia/interface'
 
 export interface HeliaFetchOptions {
   path: string
   helia: Helia
   signal?: AbortSignal
   headers?: Headers
-  origin?: string | null
+  id?: string | null
```

Comment:
:information_source:  had to rename, would be confusing for web devs  ("origin" in web security context, including `window.location`, means at minimum both protocol and the authority identifier).